bsp/chips/cc1200/cc1200.c

```diff
@@ -0,0 +1,228 @@
+/**
+\brief CC1200-specific library.
+
+\author Jonathan Munoz <jonathan.munoz@inria.fr>, August 2016.
+*/
+
+#include "board.h"
+#include "CC1200.h"
+#include "spi.h"
+
+extern const cc1200_rf_cfg_t cc1200_rf_cfg;
+
+void CC1200_spiStrobe(uint8_t strobe, cc1200_status_t* statusRead) {
```

Comment:
by convention, function names start with lowercase characters. Please rename all functions to `cc1200_*`